bootstrap/velocity/build.gradle.kts

```diff
@@ -36,6 +36,10 @@ exclude("net.kyori:adventure-nbt:*")
 // These dependencies are already present on the platform
 provided("com.velocitypowered", "velocity-api", velocityVersion)
 
+dependencies {
```

Comment:
Can we remove this? Does it need to be removed?

Reply:
No, it can't be removed it's what makes the `@Plugin` work in `GeyserVelocityPlugin`

Reply:
Isn't this annotation processor already defined in line 4?